engine/src/main/java/org/camunda/bpm/engine/impl/dmn/transformer/DecisionDefinitionHandler.java

```diff
@@ -36,10 +47,24 @@ protected DmnDecisionImpl createFromDecision(DmnElementTransformContext context,
 
     String category = context.getModelInstance().getDefinitions().getNamespace();
     decisionDefinition.setCategory(category);
-    decisionDefinition.setHistoryTimeToLive(ParseUtil.parseHistoryTimeToLive(decision.getCamundaHistoryTimeToLiveString()));
+    decisionDefinition.setHistoryTimeToLive(getEffectiveHistoryTimeToLive(decision));
```

Comment:
🙃 In my opinion this part could be simplified by not using Optionals.
```suggestion
   Integer localHistoryTimeToLive = ParseUtil.parseHistoryTimeToLive(decision.getCamundaHistoryTimeToLiveString());
    if(localHistoryTimeToLive != null) {
      decisionDefinition.setHistoryTimeToLive(localHistoryTimeToLive);
    } else {
      decisionDefinition.setHistoryTimeToLive(configuredHistoryTTL);
    }
```